backend/validators/migrations/0016_backfill_graduated_validator_profiles.py

```
@@ -0,0 +1,42 @@
+from django.db import migrations
+
+
+def backfill_graduated_validator_profiles(apps, schema_editor):
```

Ruff (0.15.20)

[warning] Unused function argument: `schema_editor`

(ARG001)